sympy/galgebra/ga.py

```diff
@@ -285,14 +285,14 @@ def make_vector(self, base):  # make a vector (grade 1)
                 if self.fct:
                     base_lst = str_combinations(base, MV.coords, rank=1, mode='__')
                     fct_lst = fct_sym_array(base_lst, MV.coords)
-                    self.obj = reduce(operator.add, tuple(map(lambda x: x[0] * x[1], zip(fct_lst, MV.blades[1]))))
+                    self.obj = reduce(operator.add, tuple([x[0] * x[1] for x in zip(fct_lst, MV.blades[1])]))
```

Comment:
There are several cases like this, I'm not going to comment on all of them. As I stated in a previous comment, this can be written as:
```
reduce(operator.add, tuple(x0 * x1 for x0, x1 in zip(fct_lst, MV.blades[1])))
```

This removes the indexing, and uses a generator expression instead of a list. No need to create a list if we're just going to turn it into a tuple right away.

Please fix all of the instances of this below as well.